urc_dashboard/README.md

```diff
@@ -0,0 +1,31 @@
+# Running the URC Dashboard
+
+Install node.js and npm using these commands:
+
+```bash
+sudo apt-get install nodejs
+sudo apt-get install npm
+```
+
+Install the dependencies:
+
+```bash
+npm install
+```
+
+Follow the instructions in [`urc_dashboard`](../urc_dashboard/README.md) package to launch Foxglove, and the websocket.
+
+Run the server:
+
+```bash
+npm start
+```
+
+This will start an application on `localhost:3000`
+
+---
+
```

Comment:
These shortcuts are not working on my computer, please look into either removing them entirely or fixing the issue